crates/uv-settings/src/lib.rs

```diff
@@ -670,3 +680,25 @@ fn parse_string_environment_variable(name: &'static str) -> Result<Option<String
         },
     }
 }
+
+/// Parse a integer environment variable.
+fn parse_integer_environment_variable(name: &'static str) -> Result<Option<u64>, Error> {
+    match std::env::var(name) {
+        Ok(v) => match v.parse::<u64>() {
```

Comment:
I would probably not nest these matches, I'd do an early return if the environment variable is not present or not unicode.

Reply:
refactored